.opspec/release/to-ghcr/check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+apk add --update docker
+
+# only continue with release if tag doesn't exist
+if docker pull ghcr.io/opctl/opctl:${version}-dind; then
+  echo "Opctl Image for version '${version}' already exists"
+  exit 1
```

Comment:
I don't think you want to exit the container non-zero cuz that will fail the operation. I think you will want to use opctl native flow logic/conditions to just skip the rest of the release in the case the release already exists.

Reply:
[docs](https://opctl.io/docs/training/flow/conditional-execution) in case that helps

Reply:
in our case I feel like we'd add a new serial block and put the condition on that and then add a container or op that lists releases from github and if the current one already exists we skip (or something like that). Maybe we add a github.releases.list op, something similar to what we do here for [repos but for releases](https://github.com/opspec-pkgs/_.op-repos.standardize/blob/6713437f2b566ba41abf43af659a5a82b5ec6c10/op.yml#L44)

Reply:
I guess another idea would just be to export a variable from the container you added something like `/isUnreleased.json: $(isUnreleased)` and condition on that instead of exiting non-zero

Reply:
done!